src/test/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventBookingManagerTest.java

```diff
@@ -11,6 +11,7 @@
 import uk.ac.cam.cl.dtg.isaac.dto.IsaacEventPageDTO;
```

Comment:
It would be good to have a few more tests dealing with what happens when users try and reserve more than they can. I've added one for you on capacity checking and fixed the bug that it caught.

The tests could do with a refactor to remove some duplicate code but they are fairly well defined and straight forward to extend.